source/clear-linux/guides/maintenance/download-verify-uncompress-windows.rst

```diff
@@ -0,0 +1,66 @@
+.. _download-verify-uncompress-windows:
+
+Download, verify, and uncompress a Clear Linux image on Windows
+###############################################################
+
+This section explains the types of |CLOSIA| images available, where to download
+them, how to verify the integrity of an image, and how to uncompress it.
+
+Alternative instructions for other operating systems are available:
+
+* :ref:`download-verify-uncompress-linux`
+* :ref:`download-verify-uncompress-mac`
+
+.. include:: types-of-cl-images.rst
+  :start-after: types-of-cl-images
+
+Download the latest Clear Linux image
+=====================================
+
+Go to the Clear Linux `image`_ repository and download the desired type of image.
+
+.. _verify-image-checksum-on-windows:
+
+Verify the integrity of the Clear Linux image
+=============================================
+
+Before you use a downloaded |CL| image, it is recommended that you verify its 
+integrity to eliminate the chance, albeit small, that it might be corrupted due 
+to any download issues.  To support this, each released |CL| image is accompanied 
```

Comment:
Do not use "this" without an appropriate noun and avoid passive voice:
To support verification, we accompany each released |CL| image with an official ...